addons/base_import_module/tests/test_cloc.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tools import cloc
+from odoo.addons.base.tests import test_cloc
+
+class TestClocFields(test_cloc.TestClocCommon):
+
+    def test_fields_from_import_module(self):
+        """
+            Check that custom computed fields installed with an imported module
+            is counted as customization
+        """
+        # Make sure previous test is still working with base_import_module installed
+        self.common_test_field_count()
```

Comment:
Can be avoided if the test inherits from `TestCloc` instead of `TestClocCommon`

Reply:
Yes but I wanted to make explicit that we are running the same test again, that's why I wrote the method common_test_field_count. Do you think this is a bad idea ?

Reply:
It works, but it's not idiomatic. An inheritance is better. A comment can still be added to explain the rationale.